src/ofbx.h

```diff
@@ -8,13 +8,8 @@ namespace ofbx
 typedef unsigned char u8;
 typedef unsigned short u16;
 typedef unsigned int u32;
-#ifdef _WIN32
-	typedef long long i64;
-	typedef unsigned long long u64;
-#else
-	typedef long i64;
-	typedef unsigned long u64;
-#endif
+typedef long long i64;
+typedef unsigned long long u64;
```

Comment:
I've removed the #ifdef _WIN32 directive since `long long` and `unsigned long long` are confirmed to be 64 bits on platforms with `_WIN32`, as verified on https://godbolt.org/z/TjK3ejeTE

```cpp
typedef unsigned char u8;
typedef unsigned short u16;
typedef unsigned int u32;
typedef long long i64;
typedef unsigned long long u64;
int main() {
    static_assert(sizeof(u8) == 1, "u8 is not 1 byte");
    static_assert(sizeof(u32) == 4, "u32 is not 4 bytes");
    static_assert(sizeof(u64) == 8, "u64 is not 8 bytes");
    static_assert(sizeof(i64) == 8, "i64 is not 8 bytes");
    return 0;
}
```

This change was needed due to issues on Android 32-bit systems. If needed, we can use more precise preprocessor directives for 32-bit platforms.

Reply:
Without the ifdef some platforms (some clang version on linux iirc) have compiler warnings when passing unsigned long long as u64, even though both are 64bit unsigned integers.

Reply:
https://github.com/nem0/OpenFBX/commit/b419075a0651a5984ea756e8df1d7abdf63e626d so the issue was on PVS.

Reply:
Got it. I've added back the ifdef to with Android support instead.